src/eos-updater.xml

```diff
@@ -7,6 +7,7 @@
     <property name="State"            type="u" access="read"/>
     <property name="UpdateID"         type="s" access="read"/>
     <property name="UpdateRefspec"    type="s" access="read"/>
+    <property name="OriginalRefspec"  type="s" access="read"/>
```

Comment:
@cosimoc Probably just my inexperience with gdbus-codegen. All the other EosUpdater state corresponds to these bus properties. I didn't know how to add a non-exported property to the object.

On the other hand, I don't know any reason this should be hidden from the client. It might be good fit them to know when the upgrade path changes from the on disk configuration.